### DISCUSSION TITLE
Word wrapping

### DISCUSSION THREAD

**Discussion by Corvette653** [f35f22fan/Ods2#5] [General]
Hi, I have one more question :)
Does your library service text wrapping?

**Comment by f35f22fan** [comment]
I'm not sure what you mean, like text wrapping inside a cell?
If so I've updated the project to include this usecase: get latest version from master, go to examples1.cc to CreateColorsAndUnderline() and uncomment:
	///stcp->SetWrapOption(true);
This will generate a file with a cell with wrapped text inside it.

**Reply by Corvette653** [reply]
Yes, that's exactly what I meant. I have cells with long text and want them to have few lines instead of one with "..."
